docs/policy.md

```diff
@@ -0,0 +1,63 @@
+---
+title: How to opt-out of sending personal data to IntelliCode services from Visual Studio
+description: Learn how to opt-out of sending personal data to IntelliCode Services in Visual Studio.
+ms.date: 09/16/2020
+ms.topic: how-to
+author: catycaldwell
+ms.author: cccaldwel
+ms.workload: intellicode
+---
+# What personal data is sent to IntelliCode services from Visual Studio
+
+> Applies to Visual Studio 2019 16.8 and newer
+
+Learn how to use group policy to manage your organization's sending personal data to the IntelliCode service from Visual Studio. First,
+we'll describe what IntelliCode collects and sends to the service. Then we'll walk
+through how to opt-out your organization.
+
+## Prerequisites
+
+The following items are required for applying group policy for IntelliCode capabilities:
+
+- A Visual Studio Admin role.
+- Visual Studio 2019 v 16.8 or newer
+
+Visual Studio IntelliCode is a suite of AI-assisted developer tools that uses machine learning to improve the inner loop developer experience in Visual Studio. Some IntelliCode capabilities require sending personal data from Visual Studio to the IntelliCode service: {e.g. Team Completions} IntelliCode [collects information about your types and type usage](https://docs.microsoft.com/visualstudio/intellicode/custom-models#data-and-privacy), computer hardware, and repository information and sends to the IntelliCode service over an HTTPS connection, without interrupting users in their tasks at the computer. The information that's collected allows IntelliCode to securely train and distribute personalized models. This document covers how to opt in or out of using IntelliCode service-backed capabilities.
```

Comment:
I think we could be more concise here by dropping the preamble about what IntelliCode is, and link the word `IntelliCode` to an introduction page instead.